Dockerfile.orch-tester

```diff
@@ -0,0 +1,35 @@
+FROM livepeerci/build AS builder
+
+RUN apt install -y wget tar
+
+WORKDIR /root
+RUN wget -qO- https://storage.googleapis.com/lp_testharness_assets/official_test_source_2s_keys_24pfs_30s_hls.tar.gz | tar xvz -C .
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+RUN go mod download
+
+ARG version
+RUN echo $version
+
+COPY . .
+
+RUN go build -ldflags="-X 'github.com/livepeer/stream-tester/model.Version=$version' -X 'github.com/livepeer/stream-tester/model.IProduction=true'" -tags mainnet cmd/orch-tester/orch_tester.go
+
+FROM debian:stretch-slim
+
+RUN apt update \
+  && apt install -y ffmpeg ca-certificates \
+  && apt clean && apt autoclean
+RUN ffmpeg -version
```

Comment:
Did you copy this bit from the `task-runner`? If so, I believe we don't need it here! And removing it will make the docker image build much faster :)

We actually only have that in the `task-runner` because not only it has a compile-time dependency on go-livepeer which requires the compiled ffmpeg bins [1], but it also calls `ffmpeg` (actually `ffprobe`) as a separate process on runtime. So we need to install it like this as well.

[1] The `task-runer` actually doesn't even use `go-livepeer` code that uses `ffmpeg`, only `drivers` lib. But there is some internal dependency leak on `go-livepeer` through the `common` package and so every package in `go-livepeer` depends on the compiled `ffmpeg` 🥲

Reply:
Actually just saw that there is an installation like this on the regular `stream-tester` docker image as well. Not sure if it is needed by the `orch-tester` binary specifically tho 🤔 

Worst case it's not a regression on build time anyway, so no worries keeping this anyway to stay on the safe side.

Reply:
Right, FFMPEG is not need at runtime, updated in 8a09f0ee0b69779386991027e8ab3eae25e9a1a6